daprdocs/content/en/contributing/docs-contrib/maintainer-guide.md

```diff
@@ -0,0 +1,384 @@
+1. Have a docs maintainer or approver review. Wait to merge the PR until release.
+
+#### Upcoming release
+
+These steps will prepare the upcoming release branch for promotion to latest release.
+
+1. Open VS Code to the Dapr docs repo.
+1. Switch to the upcoming release branch (`v1.1`) and synchronize changes:
+
+   ```bash
+   git pull upstream v1.1
+   git push origin v1.1
+   ```
+
+1. Create a new branch based off of the upcoming release:
+
+   ```bash
+   git checkout -b release_v1.1
+   ```
+
+1. In VS Code, navigate to `/daprdocs/config.toml`.
+1. Update line 1 to `baseURL - https://docs.dapr.io/`.
+1. Update the `# Versioning` section (around line 154) to display the correct versions and tags:
+
+   ```toml
+   # Versioning
+   version_menu = "v1.1 (latest)"
+   version = "v1.1"
+   archived_version = false
+   url_latest_version = "https://docs.dapr.io"
+
+   [[params.versions]]
+     version = "v1.2 (preview)"
+     url = "v1-2.docs.dapr.io"
+   [[params.versions]]
+     version = "v1.1 (latest)"
+     url = "#"
+   [[params.versions]]
+     version = "v1.0"
+     url = "https://v1-0.docs.dapr.io"
+   ```
+
+1. Navigate to `.github/workflows/website-root.yml`. 
+1. Update the branches which trigger the workflow:
+
+   ```yml
+   name: Azure Static Web App Root
+
+   on:
+     push:
+       branches:
+         - v1.1
+     pull_request:
+       types: [opened, synchronize, reopened, closed]
+       branches:
+         - v1.1
+   ```
+
+1. Navigate to `/README.md`.
+1. Update the versions table:
+
+```markdown
+| Branch                                                       | Website                    | Description                                                                                      |
+| ------------------------------------------------------------ | -------------------------- | ------------------------------------------------------------------------------------------------ |
+| [v1.1](https://github.com/dapr/docs) (primary)               | https://docs.dapr.io       | Latest Dapr release documentation. Typo fixes, clarifications, and most documentation goes here. |
+| [v1.2](https://github.com/dapr/docs/tree/v1.2) (pre-release) | https://v1-2.docs.dapr.io/ | Pre-release documentation. Doc updates that are only applicable to v1.2+ go here.                |
+```
+
+1. In VS Code, search for any `v1.0` references and replace them with `v1.1` as applicable.
+1. Commit the staged changes and push to your branch (`release_v1.1`).
+1. Open a PR from `release/v1.1` to `v1.1`.
+1. Have a docs maintainer or approver review. Wait to merge the PR until release.
+
+### Create new website for future release
+
+Next, create a new website for the future Dapr release, which will be pointed to from the latest website. To do this, you'll need to:
+
+- Deploy an Azure Static Web App.
+- Configure DNS via request from CNCF.
+
+These steps require authentication. 
+
+### Create new website for future release
```

Comment:
This has the same title as above? This looks wrong. 
Also we need to say in this section that you need MS employee access to create a new Website.